packages/client-core/src/user/menus/ProfileMenu.tsx

```diff
@@ -0,0 +1,692 @@
+                  })
+                }}
+              >
+                <Copy03Lg />
+              </button>
+            }
+            fullWidth
+          />
+        )}
+
+        {showApiKey.value && (
+          <Input
+            labelProps={{
+              text: t('user:usermenu.profile.apiKey'),
+              position: 'top'
+            }}
+            value={apiKey?.token}
+            startComponent={
+              <button className="h-4 w-4" onMouseDown={refreshApiKey}>
+                <Refresh1Lg />
+              </button>
+            }
+            endComponent={
+              <button
+                className="h-4 w-4"
+                onMouseDown={() => {
+                  navigator.clipboard.writeText(apiKey?.token)
+                  NotificationService.dispatchNotify(t('user:usermenu.profile.apiKeyCopied'), {
+                    variant: 'success'
+                  })
+                }}
+              >
+                <Copy03Lg />
+              </button>
+            }
+            fullWidth
+          />
+        )}
+
+        {!hideLogin && acceptedTOS && isGuest && enableConnect && (
+          <>
+            <Input
+              labelProps={{
+                text: getConnectText(),
+                position: 'top'
+              }}
+              placeholder={getConnectPlaceholder()}
+              state={error.value ? 'error' : undefined}
+              helperText={getErrorText()}
+              endComponent={
+                <button className="h-4 w-4" onMouseDown={handleGuestSubmit}>
+                  <Send01Lg />
+                </button>
+              }
+              fullWidth
+            />
+          </>
+        )}
+      </div>
+
+      {!isGuest && (
+        <div className="mt-5 grid w-1/2 grid-cols-1 gap-y-2 px-5">
+          <button className="flex w-full items-center justify-start gap-x-2 p-2" onClick={handleLogout}>
+            <LogIn01Lg />
+            {t('user:usermenu.profile.logout')}
+          </button>
+
+          <button
+            className="flex w-full items-center justify-start gap-x-2 p-2"
+            onClick={() => {
+              PopoverState.showPopupover(
+                <ConfirmDialog
+                  title={t('user:usermenu.profile.delete.finalDeleteConfirm')}
+                  text={t('user:usermenu.profile.delete.finalDeleteText')}
+                  onSubmit={async () => {
+                    AuthService.removeUser(userId)
+                    AuthService.logoutUser()
+                    showDeleteAccount.set(false)
+                  }}
+                />
+              )
+            }}
+          >
+            <Trash04Lg />
+            {t('user:usermenu.profile.delete.deleteAccount')}
+          </button>
+        </div>
+      )}
+
+      <hr className="mb-5 mt-5 border-[#616161]" />
+
+      {!hideLogin && acceptedTOS && enableSocial && (
+        <div className="flex w-full items-center justify-center gap-x-2">
```

Comment:
Social logins are aligned to the left in the design

Reply:
Current design does not allow user to remove OAuths individually. "Remove Social Logins" button removes all the connected OAuths. So, I improvised and made the buttons toggle the OAuth, and hence removed the "Remove Social Logins" button. 


![image](https://github.com/user-attachments/assets/0fbfd4d4-d907-40ec-a0ed-abf503a2aa94)

Reply:
Current design does not allow user to remove OAuths individually. "Remove Social Logins" button removes all the connected OAuths. So, I improvised and made the buttons toggle the OAuth, and hence removed the "Remove Social Logins" button. 


![image](https://github.com/user-attachments/assets/0fbfd4d4-d907-40ec-a0ed-abf503a2aa94)